hadoop-hdds/interface-server/src/main/resources/proto.lock

```diff
@@ -1209,6 +1209,10 @@
               {
                 "name": "INTERNAL_ERROR",
                 "integer": 29
+              },
```

Comment:
Proto.lock file change should be excluded. the `proto.lock` should represent and old state: the last released state to use it as a base for comparison.

Rebase to the latest master and you won't see the changed proto lock files (which makes it easy to add lock files accidentally)